### PR TITLE
Rewrite links that redirect to their final destination

### PR DESCRIPTION
**Pull request by ennru**

### CODE REVIEW

File: blog/_posts/2012-06-12-akka-2.0.2-released.md [hunk 1]
```diff
@@ -48,10 +48,10 @@ We are pleased to be able to announce the availability of Akka 2.0.2 which addre
 
 ## Useful Links
 
-* Akka Maven repository: [http://repo.akka.io/releases](http://repo.akka.io/releases)
+* Akka Maven repository: [https://repo.akka.io/releases](https://repo.akka.io/releases)
 * Akka binary distribution: [https://akka.io/downloads](https://akka.io/downloads)
-* Akka 2.0.2 documentation: [https://doc.akka.io/docs/akka/2.0.2/](https://doc.akka.io/docs/akka/2.0.2/)
-* Issue tracker: [http://www.assembla.com/spaces/akka/](http://www.assembla.com/spaces/akka/)
+* Akka 2.0.2 documentation: [https://doc.akka.io/docs/akka/2.0.5/](https://doc.akka.io/docs/akka/2.0.5/)
```
Comment (raboof) [inline blog/_posts/2012-06-12-akka-2.0.2-released.md:53]: hmm, perhaps we should redirect (and link) to `https://doc.akka.io/docs/akka/2.0/` instead?

File: blog/_posts/2014-02-14-akka-2.3.0-RC3-released.md [hunk 2]
```diff
@@ -27,24 +27,24 @@ When migrating an existing project from Akka 2.2.x please have a look at the [mi
...
 
 ### Other Notable Features
 
- * [Actor](https://doc.akka.io/docs/akka/2.3.0-RC3/java/lambda-actors.html) and [FSM](https://doc.akka.io/docs/akka/2.3.0-RC3/java/lambda-fsm.html) with idiomatic Java 8 lambda support, allowing the formulation of these using a syntax which is close to Scala’s pattern matching and finally making the Finite State Machine DSL usable from Java.
+ * [Actor](https://doc.akka.io/docs/akka/2.3//java/lambda-actors.html) and [FSM](https://doc.akka.io/docs/akka/2.3//java/lambda-fsm.html) with idiomatic Java 8 lambda support, allowing the formulation of these using a syntax which is close to Scala’s pattern matching and finally making the Finite State Machine DSL usable from Java.
```
Comment (raboof) [inline blog/_posts/2014-02-14-akka-2.3.0-RC3-released.md:38]:
```suggestion
 * [Actor](https://doc.akka.io/docs/akka/2.3/java/lambda-actors.html) and [FSM](https://doc.akka.io/docs/akka/2.3/java/lambda-fsm.html) with idiomatic Java 8 lambda support, allowing the formulation of these using a syntax which is close to Scala’s pattern matching and finally making the Finite State Machine DSL usable from Java.
```

File: blog/_posts/2014-02-19-akka-2.3.0-RC4-released.md [hunk 3]
```diff
@@ -26,25 +26,25 @@ When migrating an existing project from Akka 2.2.x please have a look at the [mi
...
 
 ### Other Notable Features
 
- * [Actor](https://doc.akka.io/docs/akka/2.3.0-RC4/java/lambda-actors.html) and [FSM](https://doc.akka.io/docs/akka/2.3.0-RC4/java/lambda-fsm.html) with idiomatic Java 8 lambda support, allowing the formulation of these using a syntax which is close to Scala’s pattern matching and finally making the Finite State Machine DSL usable from Java.
+ * [Actor](https://doc.akka.io/docs/akka/2.3//java/lambda-actors.html) and [FSM](https://doc.akka.io/docs/akka/2.3//java/lambda-fsm.html) with idiomatic Java 8 lambda support, allowing the formulation of these using a syntax which is close to Scala’s pattern matching and finally making the Finite State Machine DSL usable from Java.
```
Comment (raboof) [inline blog/_posts/2014-02-19-akka-2.3.0-RC4-released.md:37]:
```suggestion
 * [Actor](https://doc.akka.io/docs/akka/2.3/java/lambda-actors.html) and [FSM](https://doc.akka.io/docs/akka/2.3/java/lambda-fsm.html) with idiomatic Java 8 lambda support, allowing the formulation of these using a syntax which is close to Scala’s pattern matching and finally making the Finite State Machine DSL usable from Java.
```

File: blog/_posts/2014-03-25-akka-2.3.1-released.md [hunk 4]
```diff
@@ -24,7 +24,7 @@ This release is backwards binary compatible with version 2.3.0 which means that
 
 ### Migrating from Older Relases ###
 
-When migrating an existing project from Akka 2.2.x please have a look at our [migration guide](https://doc.akka.io/docs/akka/2.3.1/project/migration-guide-2.2.x-2.3.x.html).
+When migrating an existing project from Akka 2.2.x please have a look at our [migration guide](https://doc.akka.io/docs/akka/2.3//project/migration-guide-2.2.x-2.3.x.html).
```
Comment (raboof) [inline blog/_posts/2014-03-25-akka-2.3.1-released.md:27]:
```suggestion
When migrating an existing project from Akka 2.2.x please have a look at our [migration guide](https://doc.akka.io/docs/akka/2.3/project/migration-guide-2.2.x-2.3.x.html).
```

File: blog/_posts/2014-04-09-akka-2.3.2-released.md [hunk 5]
```diff
@@ -16,7 +16,7 @@ This release is backwards binary compatible with version 2.3.0 and 2.3.1 which m
 
 ### Migrating from Older Relases ###
 
-When migrating an existing project from Akka 2.2.x please have a look at our [migration guide](https://doc.akka.io/docs/akka/2.3.2/project/migration-guide-2.2.x-2.3.x.html).
+When migrating an existing project from Akka 2.2.x please have a look at our [migration guide](https://doc.akka.io/docs/akka/2.3//project/migration-guide-2.2.x-2.3.x.html).
```
Comment (raboof) [inline blog/_posts/2014-04-09-akka-2.3.2-released.md:19]:
```suggestion
When migrating an existing project from Akka 2.2.x please have a look at our [migration guide](https://doc.akka.io/docs/akka/2.3/project/migration-guide-2.2.x-2.3.x.html).
```

File: blog/_posts/2017-02-23-akka-http-10.0.4-released.md [hunk 6]
```diff
@@ -54,7 +54,7 @@ as an Akka Http committer!
 
 ## Binary Compatibility
 
-Akka 10.0.x is backwards binary [compatible](https://doc.akka.io/docs/akka/2.5-M1/common/binary-compatibility-rules.html)
+Akka 10.0.x is backwards binary [compatible](https://doc.akka.io/docs/akka/2.5//common/binary-compatibility-rules.html)
```
Comment (raboof) [inline blog/_posts/2017-02-23-akka-http-10.0.4-released.md:57]:
```suggestion
Akka 10.0.x is backwards binary [compatible](https://doc.akka.io/docs/akka/2.5/common/binary-compatibility-rules.html)
```

File: blog/_posts/2013-08-27-coursera-course-reactive.md [hunk 7]
```diff
@@ -18,7 +18,7 @@ Manifesto](https://www.reactivemanifesto.org/), illustrating the techniques
 using Scala’s
 [Futures](https://docs.scala-lang.org/overviews/core/futures.html), Netflix’
 [RxJava](https://github.com/Netflix/RxJava/tree/master/language-adaptors/rxjava-scala)
-and [Akka actors](https://doc.akka.io/docs/akka/2.2//general/actors.html).
+and [Akka actors](https://doc.akka.io/docs/akka/2/akka/2.0general/actors.html).
```
Comment (raboof) [inline blog/_posts/2013-08-27-coursera-course-reactive.md]: that doesn't seem right

File: blog/_posts/2015-03-05-akka-2.2.5-released.md [hunk 8]
```diff
@@ -20,7 +20,7 @@ It includes a fix concerning filebased durable mailboxes, which might have grown
 ### Migrating from Older Releases
 
 When migrating an existing project from Akka 2.1.x please have a look at our migration guide:
-https://doc.akka.io/docs/akka/2.2.5/project/migration-guide-2.1.x-2.2.x.html
+https://doc.akka.io/docs/akka/2/akka/2.0/project/migration-guide-2.1.x-2.2.x.html
```
Comment (raboof) [inline blog/_posts/2015-03-05-akka-2.2.5-released.md:23]:
```suggestion
https://doc.akka.io/docs/akka/2.2/project/migration-guide-2.1.x-2.2.x.html
```

**Comment by ennru** [comment]
I messed things up for the worse. So rolled back a bit...

**Pull request merged**